README.md

```diff
@@ -1,7 +1,15 @@
-# **tremor-www-docs**
+<p align=center><img src="https://raw.githubusercontent.com/cncf/artwork/master/projects/tremor/horizontal/color/tremor-horizontal-color.png" width='35%'></p>
+<p align=center><a href="https://landscape.cncf.io/selected=tremor">CNCF Early Stage Sandbox Project</p>
+<p align=center><a href="https://landscape.cncf.io/category=streaming-messaging&format=card-mode&grouping=category">CNCF Streaming &amp; Messaging</p>
+
+<hr>
+
+[![CII Best Practices]][bestpractices]
+[![GitHub]](LICENSE)
+[![Discord]][discord-invite]
```

Comment:
I think those three are missing the related links :)

They reference the following that need to be in the README.md to make them look/link correctly:

```markdown
[CII Best Practices]: https://bestpractices.coreinfrastructure.org/projects/4356/badge
[bestpractices]: https://bestpractices.coreinfrastructure.org/projects/4356
[GitHub]: https://img.shields.io/github/license/tremor-rs/tremor-runtime
[Discord]: https://img.shields.io/discord/752801695066488843.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2
[discord-invite]: https://bit.ly/tremor-discord

```

Reply:
Yess, I have fixed them locally didn't push them yet 😶